src/contexts/ErrorContext/ErrorContext.tsx

```diff
@@ -1,85 +1,69 @@
 import Prompt from "components/Prompt";
-import { GENERIC_ERROR_MESSAGE } from "constants/common";
-import { APError, AP_ERROR_DISCRIMINATOR } from "errors/errors";
+import { EMAIL_SUPPORT } from "constants/env";
 import { logger } from "helpers";
-import {
-  PropsWithChildren,
-  createContext,
-  useCallback,
-  useContext,
-} from "react";
+import { ReactNode } from "react";
 import { useModalContext } from "../ModalContext";
 
-type State = {
-  handleError: (
-    error: any,
-    displayMessage?: string,
-    options?: { log: boolean }
-  ) => void;
-};
+function parseError(error: unknown): string | undefined {
```

Comment:
Maybe wrap these into `useCallback`? That way we can safely use them in dependency arrays, as surely there'll be a need.

Reply:
looking ahead, I don't see yet the need for it to be used prob in `useMemo` or `useEffect` - as this helper is only internal to `errors/ErrorContext` - but we could sure wrap as the need arises.